### DISCUSSION TITLE
Has anyone got it to work with Ntrip in Australia?

### DISCUSSION THREAD

**Discussion by rbtchun3** [semuconsulting/pygnssutils#44] [General]
Hi!

I tried to get it working with [Australia Ntrip](https://gnss.ga.gov.au/stream)
I get the sourcetable when leaving mountpoint empty

but I get nothing when running this
```gnssntripclient server=ntrip.data.gnss.ga.gov.au ggamode=1 refla=-23.94 reflon=150.27 mountpoint=DIXL00AUS0 ggainterval=60 user=<registered name> pasword=<registered pwd> ```

Any opinion is welcome! Thanks!

**Comment by semuadmin** [comment]
Hi @rbtchun3 

You haven't specified which version of pygnssutils you're using, but all versions since 1.0.2 use a different command line syntax. The correct syntax in your case would be:

```
gnssntripclient --server ntrip.data.gnss.ga.gov.au --ggamode 1 --reflat -23.94 --reflon 150.27 --mountpoint DIXL00AUS0 --ggainterval 60 --ntripuser user --ntrippassword password
```

I would recommend you upgrade to  the latest version of the pygnssutils library if you haven't already (v1.0.15), and then try the syntax above. For help with the available command line options, type `gnssntripclient -h`.

Let me know how you get on.

**Reply by rbtchun3** [reply]
thanks for the quick response. I can't upgrade at this moment due to our dependency management issues. But I got it to work without changing anything
